app/pods/publications/publication/route.js

```diff
@@ -80,13 +80,22 @@ export default class PublicationRoute extends Route.extend(AuthenticatedRouteMix
     });
   }
 
+  async afterModel(model) {
+    this.urgencyLevel = await model.publicationFlow.urgencyLevel;
+  }
+
   /* eslint-disable id-length,no-unused-vars */
   resetController(controller, _, transition) {
     controller.publicationNotAfterTranslationForPublication = false;
     controller.publicationNotAfterTranslationForTranslation = false;
     controller.numberIsAlreadyUsed = false;
   }
 
+  async setupController(controller) {
+    super.setupController(...arguments);
+    controller.urgencyLevel = this.urgencyLevel;
```

Comment:
Dit is wat vreemd om te doen. Waarom wil je dit als aparte property op de controller definieren en gebruik je niet `model.publicationFlow.urgencyLevel`? 
In de action-handler `setUrgencyLevel` ga je `this.model.publicationFlow.urgencyLevel` aanpassen, waardoor `controller.urgencyLevel` niet meer de correcte waarde bevat.

Reply:
Offline met @MikiDi besproken. Dit is om te vermijden dat promises als argument doorgegeven worden aan een component. Er moet dus enkel gezorgd worden dat ook `controller.urgencyLevel` aangepast wordt in de action-handler `setUrgencyLevel`.